python/CMakeLists.txt

```diff
@@ -0,0 +1,118 @@
+	pointmatcher/impls/transformationcheckers_impl.cpp
+	pointmatcher/impl.cpp
+	pointmatcher/inspector.cpp
+	pointmatcher/io.cpp
+	pointmatcher/matcher.cpp
+	pointmatcher/matches.cpp
+	pointmatcher/outlierfilter.cpp
+	pointmatcher/outlierfilters.cpp
+	pointmatcher/pointmatcher.cpp
+	pointmatcher/transformation.cpp
+	pointmatcher/transformations.cpp
+	pointmatcher/transformationchecker.cpp
+	pointmatcher/transformationcheckers.cpp
+
+	#pointmatchersupport module
+	pointmatchersupport/bibliography.cpp
+	pointmatchersupport/logger.cpp
+	pointmatchersupport/logger_impl.cpp
+	pointmatchersupport/parametrizable.cpp
+	pointmatchersupport/registrars/datapointsfilter_registrar.cpp
+	pointmatchersupport/registrars/errorminimizer_registrar.cpp
+	pointmatchersupport/registrars/inspector_registrar.cpp
+	pointmatchersupport/registrars/logger_registrar.cpp
+	pointmatchersupport/registrars/matcher_registrar.cpp
+	pointmatchersupport/registrars/outlierfilter_registrar.cpp
+	pointmatchersupport/registrars/transformation_registrar.cpp
+	pointmatchersupport/registrars/transformationchecker_registrar.cpp
+	pointmatchersupport/registrar.cpp
+
+	#errorminimizers module
+	errorminimizers/identity.cpp
+	errorminimizers/pointtoplane.cpp
+	errorminimizers/pointtoplanewithcov.cpp
+	errorminimizers/pointtopoint.cpp
+	errorminimizers/pointtopointsimilarity.cpp
+	errorminimizers/pointtopointwithcov.cpp
+
+	#datapointfilters module
+	datapointsfilters/boundingbox.cpp
+	datapointsfilters/covariancesampling.cpp
+	datapointsfilters/cutatdescriptorthreshold.cpp
+	datapointsfilters/distancelimit.cpp
+	datapointsfilters/ellipsoids.cpp
+	datapointsfilters/fixstepsampling.cpp
+	datapointsfilters/gestalt.cpp
+	datapointsfilters/identity.cpp
+	datapointsfilters/incidenceangle.cpp
+	datapointsfilters/maxdensity.cpp
+	datapointsfilters/maxpointcount.cpp
+	datapointsfilters/maxquantileonaxis.cpp
+	datapointsfilters/normalspace.cpp
+	datapointsfilters/observationdirection.cpp
+	datapointsfilters/octreegrid.cpp
+	datapointsfilters/orientnormals.cpp
+	datapointsfilters/randomsampling.cpp
+	datapointsfilters/removenan.cpp
+	datapointsfilters/removesensorbias.cpp
+	datapointsfilters/samplingsurfacenormal.cpp
+	datapointsfilters/shadow.cpp
+	datapointsfilters/simplesensornoise.cpp
+	datapointsfilters/sphericality.cpp
+	datapointsfilters/surfacenormal.cpp
+
+	modules/pointmatcher_module.cpp
+	modules/pointmatchersupport_module.cpp
+	modules/datapointsfilters_module.cpp
+	modules/errorminimizers_module.cpp
+
+	# main module
+	pypointmatcher.cpp)
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}
+					${CMAKE_SOURCE_DIR}/pointmatcher
+					${CMAKE_SOURCE_DIR}/pointmatcher/DataPointsFilters
+					${CMAKE_SOURCE_DIR}/pointmatcher/DataPointsFilters/utils
+					${CMAKE_SOURCE_DIR}/pointmatcher/ErrorMinimizers)
+
+if (pybind11_FOUND)
+	message(STATUS "pybind11 v${pybind11_VERSION}")
+
+	pybind11_add_module(pypointmatcher ${PYBIND11_SOURCES})
+
+	target_link_libraries(pypointmatcher
+						  PRIVATE
```

Comment:
Is this file written using spaces or tabs? It is not displayed uniformly on my browser.

Reply:
I used tabs. I have the same issue in my browser too, but the display were fine in my IDE. I've changed my reformat settings for cmake files and it will be fix in a future commit. 👍🏻